include/fmt/core.h

```diff
@@ -64,7 +64,8 @@
 #  define FMT_HAS_FEATURE(x) 0
 #endif
 
-#if defined(__has_include) && !defined(__INTELLISENSE__) && \
+#if defined(__has_include) &&                                               \
+    (!defined(__INTELLISENSE__) || (!FMT_MSC_VER || FMT_MSC_VER > 1900)) && \
```

Comment:
I don't think we need the `!FMT_MSC_VER` check here because it will be covered by `!defined(__INTELLISENSE__)`.

Reply:
You are right, `!FMT_MSC_VER` is a redundant check. I have removed it.